recipes/gegca/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gegca" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gegca-{{ version }}.tar.gz
+  sha256: 6b8f9e3d384c3b5b30d3601f5501f500883dd3cc8138c1febbcb1d949d9c0f3d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - pandas
+  run_constrained:
```

Comment:
This is a no-op. What do you want to achieve with these constraints?

Reply:
Nothing, they just came from what Grayskull produced and I figured they didn't hurt so I left them. I'm happy to delete them.

Reply:
Nothing, they just came from what Grayskull produced and I figured they didn't hurt so I left them. I'm happy to delete them.